Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -164,6 +201,11 @@ const StatusBar = React.createClass({
       StatusBarManager.setNetworkActivityIndicatorVisible(visible);
     },
 
+    /**
+     * Set the background color for the status bar
+     * @param {string} color Background color.
+     * @param {boolean=} animated Animate the style change.
```

Comment:
Commented pretty much the same haha.

This is what I did when I implemented the doc for component methods in react-docgen so it's definetely doable. Something like:

1. Parse ast with recast.
2. Find methods and the corresponding entry in the jsdoc parsed object.
3. Find flow types for parameters / return value using docgen helpers.
4. Update the jsdoc doc object.